api/resolvers/wallet.js

```diff
@@ -663,16 +666,34 @@ const resolvers = {
 
 export default injectResolvers(resolvers)
 
-export const addWalletLog = async ({ wallet, level, message }, { models }) => {
-  try {
-    await models.walletLog.create({ data: { userId: wallet.userId, wallet: wallet.type, level, message } })
-  } catch (err) {
-    console.error('error creating wallet log:', err)
+export const walletLogger = ({ wallet, models }) => {
+  // server implementation of wallet logger interface on client
+  const log = (level) => async (message, context = {}) => {
+    try {
+      await models.walletLog.create({
```

Comment:
no `console.log` here since I think wallets logs inside the server logs would be more noise than signal